### PR TITLE
APIS-6884: integration test update

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -64,11 +60,6 @@ lazy val microservice = Project(appName, file("."))
     Test / unmanagedSourceDirectories += (baseDirectory.value / "test-common"),
     inConfig(Test)(BloopDefaults.configSettings)
   )
-  .settings(
-    IntegrationTest / testOptions ++= Seq(Tests.Argument(TestFrameworks.ScalaTest, "-eT")),
-    IntegrationTest / unmanagedSourceDirectories += (baseDirectory.value / "test-common"),
-    inConfig(IntegrationTest)(BloopDefaults.configSettings)
```
Comment (AndySpaven) [inline build.sbt:70]: line 65 is also unnecessary

**Review by johnsgp — DISMISSED** [review]

File: build.sbt [hunk 2]
```diff
@@ -79,11 +70,21 @@ lazy val microservice = Project(appName, file("."))
...
+    DefaultBuildSettings.itSettings()
+  )
 
+commands ++= Seq(
+  Command.command("cleanAll") { state => "clean" :: "it/clean" :: state },
+  Command.command("fmtAll") { state => "scalafmtAll" :: "it/scalafmtAll" :: state },
+  Command.command("fixAll") { state => "scalafixAll" :: "it/scalafixAll" :: state },
+  Command.command("testAll") { state => "test" :: "it/test" :: state },
+  Command.command("run-all-tests") { state => "testAll" :: state },
   Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
```
Comment (AndySpaven) [inline build.sbt:88]: use cleanAll (non standard entry)

File: build.sbt [hunk 3]
```diff
@@ -79,11 +70,21 @@ lazy val microservice = Project(appName, file("."))
...
-
-  // Coverage does not need compile !
   Command.command("pre-commit") { state => "clean" :: "scalafmtAll" :: "scalafixAll" :: "coverage" :: "run-all-tests" :: "coverageOff" :: "coverageAggregate" :: state }
```
Comment (AndySpaven) [inline build.sbt:89]: see others for callAll and run-all-tests =>testAll

File: build.sbt (hunk 2 above)
Comment (mattclark-zerogravit) [reply build.sbt]: sorted

**Review by AndySpaven — APPROVED** [review]

**Pull request merged**